impeller/entity/contents/filters/blend_filter_contents.h

```diff
@@ -78,6 +78,20 @@ class BlendFilterContents : public ColorFilterContents {
       std::optional<Scalar> alpha,
       ColorFilterContents::AbsorbOpacity absorb_opacity) const;
 
+  /// @brief Implements the advanced blends filters in terms of the framebuffer
+  /// blend filters.
+  ///
+  /// This allows a substantial reduction in the number of bootstrapped filters.
+  std::optional<Entity> CreateFramebufferAdvancedBlend(
```

Comment:
Also document the `SupportsFramebufferFetch` requirement?

Reply:
Framebuffer advanced blends vs the other kind (pipeline blend?) I think are our own inventions of terminology (which totally make sense). But it would be good to point it out either here or in a glossary.